Added a simple test of temperature monitor for the system.

thefurfel opened the pull request:
These features are not testable because they need to be run in real Raspberry Pi environment.
We need to implement system calls for example to control wireless networks, etc.
Here I included an example of getting CPU temperature (hope it works).
Throw suggestions on how to organize it better.

pfoof committed:
Added a simple test of temperature monitor for the system.

stanbar committed:
add ktor serverhttp alternative

Took 33 minutes

stanbar commented:
I refactored it into dependency injection pattern, formatted code (Please use Ctrl+Alt+L), and added  "error" status. 
I think that some simple unit tests would be great here.

PatrykMilewski committed:
Separated level of abstractions by splitting run method in TemperatureMonitor

Added missing (needs verification) NotNull annotation to parameter in Manager class

Fixed one javadoc problem, removed protected access modifier since it was reduntant and changed return path in getCpuTemperature method to ?: operator for simplicity

PatrykMilewski commented:
I changed a few things, since it was easier to change them than review @Furfel

stanbar reviewed (approved):
My small suggestions would be to change name from `getActualTemperature(Process process)` to `parseTemperatureFrom(Process process)` and extract  `Pattern pattern = Pattern.compile("^[0-9]+$");` to class level field, since it doesn't need to be compiled every time, but not a big deal.

PatrykMilewski committed:
Added Stasbar suggestions from PR#27
